sql/wsrep_mysqld.cc

```diff
@@ -2996,6 +2996,11 @@ void wsrep_handle_mdl_conflict(MDL_context *requestor_ctx,
       THD_STAGE_INFO(request_thd, stage_waiting_ddl);
       ticket->wsrep_report(wsrep_debug);
       mysql_mutex_unlock(&granted_thd->LOCK_thd_data);
+      if (granted_thd->current_backup_stage != BACKUP_FINISHED &&
```

Comment:
I'm surprised to find that there is no changes to mariabackup, why is that? I think backup operation should be aborted with clear error message if we here abort it.